python/lsst/afw/fits/fits.cc

```diff
@@ -22,21 +22,185 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
 
 #include "lsst/pex/exceptions/Exception.h"
 #include "lsst/pex/exceptions/Runtime.h"
 #include "lsst/pex/exceptions/python/Exception.h"
+#include "lsst/daf/base.h"
+#include "lsst/afw/image/Image.h"
 
 #include "lsst/afw/fits.h"
 
 namespace py = pybind11;
 
 using namespace lsst::afw::fits;
+using namespace lsst::afw::fits::detail;
+using namespace pybind11::literals;
+
+
+void definePixelArray(py::module & mod) {
+    // Very minimal because this carries internal details and shouldn't be needed from python
+    py::class_<detail::PixelArrayBase, std::shared_ptr<detail::PixelArrayBase>> cls(mod, "PixelArrayBase");
+    cls.def("getNum", &PixelArrayBase::getNum);
```

Comment:
I thought it might be helpful, and I didn't know if the world would end if someone got one and the system didn't know how to interpret it.
I'll remove it.